requirements/quality.txt

```diff
@@ -183,15 +183,15 @@ mccabe==0.7.0
     # via pylint
 mock==4.0.3
     # via -r requirements/test.txt
-newrelic==7.16.0.178
+newrelic==8.0.0.179
```

Comment:
Dropped support for python 3.6.
https://github.com/newrelic/newrelic-python-agent/releases/tag/v8.0.0.179